services/web-app/components/dashboard/dashboard.module.scss

```diff
@@ -29,3 +29,99 @@
     padding-left: $spacing-05;
   }
 }
+
+.title {
+  padding-bottom: $spacing-02;
+  color: $text-secondary;
+  @include type-style('body-short-01');
+}
+
+.subtitle {
+  padding-bottom: $spacing-07;
+  color: $text-primary;
+  @include type-style('body-short-02');
+}
+
+.maintainer {
+  padding-bottom: $spacing-06;
+  color: $text-primary;
+  @include type-style('expressive-heading-04');
+}
+
+.meta {
+  color: $text-secondary;
+  @include type-style('label-01');
+
+  &--absolute {
+    position: absolute;
+    bottom: $spacing-04;
+    left: $spacing-04;
+  }
+}
+
+.meta-item {
+  display: inline;
+
+  &:not(:first-child) {
+    margin-left: $spacing-03;
+  }
+}
+
+.meta-icon {
+  position: relative;
+  top: 3px;
+  margin-right: $spacing-02;
+}
+
+.meta-status {
+  color: $text-primary;
+  @include type-style('body-short-02');
+}
+
+.library {
+  padding-bottom: $spacing-07;
+}
+
+.library-title {
+  text-decoration: none;
+  @include type-style('body-short-01');
+}
+
+.design-kit {
```

Comment:
You should be able to remove all of these design kit styles when it becomes a button. I'd review if that's a button or dropdown with @jeanservaas.